app/controllers/articles_controller.rb

```diff
@@ -18,4 +16,13 @@ def create
   def article_params
     params.require(:article).permit(:title, :body)
   end
+
+
+  def error_message(article)
+    output = ''
+    article.errors.full_messages.each do |error|
+      output += "#{error} "
```

Comment:
Please take a second look at this and review it in the browser. It is not a blocker imo but will this not mean that the error messages (if there are more than one) end up as one single string without spaces between them?

Reply:
It will end up as one string but with a space between them! 
We didn't really know how to make use of multiple flashes with out making the first one disappear.
Would love some pointers! 
 
We decided to go with it for now and when we get to the styling we would make it look better.